aries-site/src/pages/templates/internationalization.mdx

```diff
@@ -0,0 +1,97 @@
+
+## Overview
+
+Internationalizion (I18N) and Localization (L10N) are important for web content
+and applications which are intended to be used by global users. It's even important
+for users in a single location who simply speak different languages.
+
+Internationalization is the process of making your application tailor its
```

Comment:
```suggestion
Internationalization is the process of tailoring application
```

Reply:
Thanks @taysea ! I commited most of the wording tweaks.